Source/Microsoft.Teams.Apps.GoalTracker/ClientApp/src/i18n.ts

```diff
@@ -7,17 +7,22 @@
 import i18n from "i18next";
 import { initReactI18next } from "react-i18next";
 import Backend from 'i18next-xhr-backend';
+import * as microsoftTeams from "@microsoft/teams-js";
+let locale = "en-US";
+microsoftTeams.initialize();
+microsoftTeams.getContext((context: microsoftTeams.Context) => {
+    i18n.changeLanguage(context.locale!);
+});
 
 i18n
     .use(Backend)
     .use(initReactI18next) // passes i18n down to react-i18next
     .init({
-        lng: window.navigator.language,
-        fallbackLng: 'en-US',
+        lng: locale,
+        fallbackLng: locale,
```

Comment:
Is this one-level fallback, or if `fr` is available, does it load that for `fr-fr` and `fr-ca`?